Packs/Netmiko/Integrations/Netmiko/README.md

```diff
@@ -1,120 +1,118 @@
-
-
 ## Netmiko SSH module integration
 
-This integration provides ssh-based access to network devices, servers, and other appliances that support this method of configuration. For a complete list of supported platforms, visit the following URL:
+  This integration provides ssh-based access to network devices, servers, and other appliances that support this method of configuration. For a complete list of supported platforms, please visit the below URL:
 
   [Netmiko Platforms.md on Github](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md)
 
-## Configure Netmiko Integration in Cortex XSOAR
+## Configuration Parameters
 
- 1. Navigate to **Settings** > **Integrations**.
- 2. Search for **Netmiko**.
- 3. Click **Add instance** to create and configure a new integration instance.
-	 - *Name*: a name for the integration instance.
-	 - *Platform*: the platform identifier taken from the above SSH or Telnet platform name lists (e.g., linux_ssh, paloalto_panos, etc.).
-	 - *Hostname*: The IP address, hostname, or FQDN for the device to connect to via SSH.
-	 - *Port*: The port to connect to via SSH.
-	 - *Credentials*: The username/password, or XSOAR credential object, to be used for the connection.
-	 **NOTE**: Platform names are taken from the supported [SSH](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md#supported-ssh-device_type-values) or [Telnet](https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md#supported-telnet-device_type-values) device type lists on GitHub.
+ - ***Name*** - Integration instance name
+ - ***Platform*** - The Netmiko-specific platform name
+ - ***Hostname*** - The IP address, Hostname, or FQDN to connect to over SSH
+ - ***Port*** - The port to use for the SSH connection
+ - ***Credentials*** - The credentials should be the same as the Tanium client.
+ - ***Override the default timeout value*** - Override the default read timeout value used for a SSH connection (useful for slow-responding devices). 
```

Comment:
Not sure why this section is needed since it is repeated in the next section. I would remove this section.
Also, the last parameter in this section is  Override the default timeout value, but in the next section it is  TimeoutOverride. Which is correct?

Reply:
I originally had it as TimeoutOverride, but was advised by Rotem to change it to the above.